src/styles/globals.css

```diff
@@ -0,0 +1,36 @@
+html,
```

Comment:
please create `styles/reset.scss` file and import it here

https://meyerweb.com/eric/tools/css/reset/